### ISSUE TITLE
Grouped Lights needs displayType: slider or they get a Switch everytime

### ISSUE THREAD

**Issue by jimmyhawkin** [DBuit/light-popup-card#68]
Grouplights that before latest update get a Button insted of a slider everytime.

And they worked fine before update. Now i had to setup displayType: slider on all my group lights in order for it to be detected as a dimmable light. 

I dono if thats by design. But feels off.

**Comment by theglus** [comment]
@jimmyhawkin I agree, it feels off for `auto` to select a button instead of a slider for light groups.

Thanks for letting me know the fix, I was racking my brain trying to figure out what happened.
